python/shotgun_desktop/paths.py

```diff
@@ -98,11 +99,19 @@ def get_pipeline_configuration_info(connection):
     )
     logger.debug(pprint.pformat(pcs))
 
-    if len(pcs) == 0:
-        pc = None
-    else:
-        # Pick the last result. See the big comment before the Shotgun query to understand.
-        pc = pcs[-1]
+    pc = None
+    if len(pcs) != 0:
```

Comment:
Or > 0. I think it's best to simply use `if len(pcs):`

But I like that you initialize `pc` instead of the if/else.